bbp/nse/blue_repair_sdk/default.nix

```diff
@@ -0,0 +1,25 @@
+{ fetchgitExternal, stdenv, boost, cmake, pkgconfig, gsl, bbpsdk, hdf5-cpp, zlib }:
+
+stdenv.mkDerivation rec {
+  name = "BlueRepairSDK-${version}";
```

Comment:
It does not matter much. I just try usually to keep them consistent between variable name and package name. but this is your package, keep it like that if you want :)